Source/Relativity.Transfer.SDK.Samples.Core/Attributes/SampleOrder.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Relativity.Transfer.SDK.Samples.Core.Attributes;
+
+public enum SampleOrder
```

Comment:
```suggestion
internal enum SampleOrder
```

Reply:
It can't be internal because I use it into Repository layer when assigning the Sample attribute to classes. The SampleOrder is in Core layer.